xrpl/models/requests/get_aggregate_price.py

```diff
@@ -53,7 +51,7 @@ def _get_errors(self: GetAggregatePrice) -> Dict[str, str]:
 
 @require_kwargs_on_init
 @dataclass(frozen=True)
-class Oracle(TypedDict):
+class OracleEntry:
```

Comment:
Would this be confusing given that the other `Oracle` class is in `LedgerEntry`?

Naming is hard 😕

Reply:
Yeah not sure 😅 We could rename it to `OracleParamEntry` or `OracleParam`?

Reply:
Looks like they're identical, so why not just have one object?

Reply:
Not quite identical in types: one has `oracle_document_id` set to `Union[str, int]` and the other has `int`. But I guess using `Union[str, int]` would be sufficient.

Reply:
I'm guessing `Union[str, int]` is probably correct for the RPC too, tbh.

Reply:
https://github.com/XRPLF/rippled/issues/5014

Reply:
Done in https://github.com/XRPLF/xrpl-py/pull/704/commits/56f1260af99e5101de24eb6c31d6c60f4caa4a57